forloop_modules/queries/db_model_templates.py

```diff
@@ -104,12 +106,34 @@ class DeleteUidObject(BaseModel):
 
 
 class APITrigger(BaseModel):
-    trigger_name: str = ""
-    machine_uid: str = ""
-    pipeline_uid: str = ""
-    first_run_datetime: datetime.datetime
-    frequency: int = ""
-    project_uid: str = "0"
+    name: Optional[str] = None
+    first_run: datetime.datetime
```

Comment:
could be also Bool based on the name - first_run_datetime or first_run_utc is better name